bundle/manifests/windows-machine-config-operator.clusterserviceversion.yaml

```diff
@@ -350,6 +350,7 @@ spec:
               containers:
               - args:
                 - --debugLogging
+                - --runtime=docker
```

Comment:
This is a feature flag and should be a bool. We either want to use docker or we dont want to use docker, making it a string is adding complexity.

Reply:
The string flag is added keeping in mind the proposed enhancement to use --runtime option. And having a bool for runtime true/false does not help representing docker/containerd option.

Reply:
Thanks for pointing that out. Since its in the enhancement thats fine.

Reply:
Initially started with boolean in the proposal document and then moved string for more runtime options ( what if we plan to support CRIO in 6.0, just a thought )  and keep the implementation open to accommodate future change. But we know it will not happen. We can move it to boolean.

Reply:
>what if we plan to support CRIO in 6.0, just a thought

Not a valid argument 😄 CRI-O does not have Windows support and there are no plans to add support for CRI-O as you called out in the enhancement. Anyhow this is a temporary flag, so whatever is easier.